scripts/token_checker.py

```diff
@@ -23,7 +23,8 @@ def __init__(self):
                             "fantom": "ftm-fantom",
                             "celo": "celo-celo",
                             "base": "base-base",
-                            "zksync": "zksync-zksync"
+                            "zksync": "zksync-zksync",
+                            "zora": "zora-zora"
```

Comment:
`zora` may not be supported in the coinspaprika list yet:
https://api.coinpaprika.com/v1/contracts/

```suggestion
                            "zksync": "zksync-zksync"
```

Reply:
hm, good catch, let's commit this suggestion and see the logs output